app/mailers/payment_mailer.rb

```diff
@@ -17,12 +17,14 @@ def authorize_payment(payment)
 
   def authorization_required(payment)
     @payment = payment
-    shop_owner = @payment.order.distributor.owner
+    @order = @payment.order
+    shop_owner = @order.distributor.owner
     subject = I18n.t('spree.payment_mailer.authorization_required.subject',
-                     order: @payment.order)
+                     order: @order)
     I18n.with_locale valid_locale(shop_owner) do
       mail(to: shop_owner.email,
-           subject:)
+           subject:,
+           reply_to: @order.email)
```

Comment:
In Ruby, the `@var` notation refers to instance variables that are available beyond the method in which they are defined. So in controllers they are often defined to be used in the views. And here, `@payment` can be used in the email template.

The new variable `@order` however, is not used in the template. It's only used for translating the subject and the reply_to field. So you can turn this into a normal local variable. Other developers then know that it's not used beyond this scope.

I'll add a commit to show you what I mean.